src/Resources/js/views/mealIndex.ts

```diff
@@ -1,39 +1,54 @@
 import {ParticipationPreToggleHandler} from "../modules/participation-pre-toggle-handler";
 import {ParticipationToggleHandler} from "../modules/participation-toggle-handler";
-import {ParticipationCountUpdateHandler} from "../modules/participation-count-update-handler";
 import {CombinedMealDialog, SerializedFormData} from "../modules/combined-meal-dialog";
 import {ParticipationRequest, ParticipationRequestHandler} from "../modules/participation-request-handler";
-import {UpdateOffersHandler} from "../modules/update-offers-handler";
 import {ParticipationResponse} from "../modules/participation-response-handler";
 import {CombinedMealService} from "../modules/combined-meal-service";
+import {MercureSubscriber} from "../modules/subscriber/mercure-subscriber";
+import {MealOfferUpdate, MealOfferUpdateHandler} from "../modules/meal-offer-update-handler";
+import {ParticipationUpdateHandler} from "../modules/participation-update-handler";
+import {SlotAllocationUpdateHandler} from "../modules/slot-allocation-update-handler";
+
+interface UpdateResponse extends ParticipationResponse {
+    bookedDishSlugs: string[];
+}
 
 export default class MealIndexView {
     participationPreToggleHandler: ParticipationPreToggleHandler;
     $participationCheckboxes: JQuery;
 
     constructor() {
-        this.updateSlots();
-        setInterval(this.updateSlots, 3000);
-
         this.$participationCheckboxes = $('.meals-list .meal .participation-checkbox');
         this.initEvents();
 
         if (this.$participationCheckboxes.length > 0) {
             let participationToggleHandler = new ParticipationToggleHandler(this.$participationCheckboxes);
             this.participationPreToggleHandler = new ParticipationPreToggleHandler(participationToggleHandler);
-
-            new ParticipationCountUpdateHandler(this.$participationCheckboxes);
-            new UpdateOffersHandler();
         }
+
+        let messageSubscriber = new MercureSubscriber($('.weeks').data('msgSubscribeUrl'));
```

Comment:
can be written without declaration of variable

Reply:
Can you give an example?